# estimated target UAV state
bool tar_OK
bool is_laser_measured

float64 tar_x
float64 tar_y
float64 tar_z
float64 tar_vx
float64 tar_vy

float64 est_dis

